src/colophon.adoc

```diff
@@ -13,43 +14,105 @@ The document contains the following versions of the RISC-V ISA modules:
 [%autowidth,float="center",align="center",cols="^,<,^",options="header"]
 |===
 |Base |Version |Status
-|*RVWMO* |*2.0* |*Ratified*
 |*RV32I* |*2.1* |*Ratified*
-|*RV64I* |*2.1* |*Ratified*
 |*RV32E* |*2.0* |*Ratified*
 |*RV64E* |*2.0* |*Ratified*
+|*RV64I* |*2.1* |*Ratified*
 |_RV128I_ |_1.7_ |_Draft_
 
 h|Extension h|Version h|Status
 
+|*Zifencei* |*2.0* |*Ratified*
+|*Zicsr* |*2.0* |*Ratified*
+|*Zicntr* |*2.0* |*Ratified*
+|*Zihintntl* |*1.0* |*Ratified*
+|*Zihintpause* |*2.0* |*Ratified*
+|_Zimop_ | _0.1_ | _Draft_
```

Comment:
Also, please update this to Ratified/1.0.  (This happened after you opened the PR; it wasn't a mistake on your part.)

Reply:
Done.